### DISCUSSION TITLE
Channel is listed in the csv, but not able to pull stream in xteve

### DISCUSSION THREAD

**Discussion by s66jones** [iptv-org/community#127] [Q&A] [answered]
I see the (sports) channel I am looking for listed in the channel csv file, but when pulling https://iptv-org.github.io/iptv/categories/sports.m3u xteve does not pull the stream. So I tested https://iptv-org.github.io/iptv/index.m3u and still dont see the stream. https://iptv.maheshwarravuri.com/ and http://tvpeer.github.io/ don't list the channel ether. So my question is, why not? How come its listed in the daily channel csv but not an available stream?

**Comment by BellezaEmporium** [comment, marked as answer]
Most of the sports channels were DMCAd, and, as we told before, what you see in the CSV is **not** representative of all channels we have in our playlists.

We have more channels in the CSV file because we want to build usable data for everyone, but also to make sure that if we have an EPG for those channels, we have all of them listed.

**Reply by s66jones** [reply]
Thank you for the explanation.
